src/test/java/task24/AgeVerificationTest.java

```diff
@@ -0,0 +1,21 @@
+package task24;
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class AgeVerificationTest {
```

Comment:
Nie potrzebujesz drugiej klasy testującej dokładnie to samo co powyższa. Klasa z pakietu `project;` może zostać usunięta.